aspnetcore/blazor/tutorials/movie-database-app/part-1.md

```diff
@@ -0,0 +1,435 @@
+
+At the end of this tutorial, you'll have an interactive Blazor Web App that manages a database of movies.
+
+## Prerequisites
+
+<!-- REVIEWER NOTE
+
+Let's not cross-link to specific tooling versions to 
+avoid content versioning and save on doc maintenance 
+costs.
+
+General reminder: As usual for tutorials, there's 
+minimal cross-linking to reference docs in the text 
+of the series to avoid distracting the reader. We do 
+link to API docs in the text. Reference doc 
+cross-links go into Additional Resources sections at 
+the bottoms of the articles.
+
+-->
+
+:::zone pivot="vs"
+
+<!-- NOTE TO SELF: Confirm this is the link for continued use -->
+
+[Visual Studio (latest release)](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2022) with the **ASP.NET and web development** workload
+
+:::zone-end
+
+:::zone pivot="vsc"
+
+Latest releases of:
+
+* [Visual Studio Code](https://code.visualstudio.com/download)
+* [C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+* [.NET SDK](https://dotnet.microsoft.com/download/dotnet)
+
+The Visual Studio Code (VS Code) instructions for ASP.NET Core development in this tutorial use the [.NET CLI](/dotnet/core/tools/), which is part of the .NET SDK. .NET CLI commands are issued in VS Code's integrated [**Terminal**](https://code.visualstudio.com/docs/editor/integrated-terminal), which is a [PowerShell command shell](/powershell/).
+
+:::zone-end
+
+:::zone pivot="cli"
+
+[.NET SDK (latest release)](https://dotnet.microsoft.com/download/dotnet)
+
+The [.NET CLI](/dotnet/core/tools/) is part of the .NET SDK. To issue a command that affects the project, open the command shell to the project's root folder to execute the command.
+
+<!-- REVIEWER NOTE
+
+I think a PS command shell would be fine to use
+as well. If you want me to add a remark here that 
+the dev can use PowerShell for the tutorial, let 
+me know. A nice feature of PS is that the dev 
+can right-click the project and select 
+'Open in Terminal' to get a PS command shell 
+for the app without having to fiddle around with
+a change directory step.
+
+-->
+
+:::zone-end
+
+Patch releases in .NET roll-forward automatically, so if you have 8.0.0 installed, the app automatically obtains NuGet packages, builds, runs locally, and publishes with the latest 8.0.x patch release.
+
+We recommend avoiding preview tooling and preview framework releases when using .NET and Blazor for the first time, as features may be unstable while the tooling and framework are under construction for the upcoming release. 
+
```

Comment:
Just noting in passing that this has broken folks in the past and generated doc issues (💸). 

WRT the patch release roll-forward behavior, it has also led some to some unnecessary doc costs because devs see the sample app without the latest patch packages explicitly referenced. They open an issue/PR that then must be dealt with. Since we're killing the line here, I'm going to make a remark about this the sample app's README file to discourage issues/PRs for it.